### PR TITLE
Consistent naming of FNR and FPR

### PR DESCRIPTION
**Pull request by gordonkoehn**
> In the original SCITE paper stands for FPR (false positive rate) and stands for FNR (false negative rate).
>  
>  It'd be good to go through the code and see whether the naming is consistent. I'd also perhaps suggest dropping 
>  and notation entirely and use fpr and fnr instead.

see / resolves Issue #30

### CODE REVIEW

File: src/pyggdrasil/tree_inference/_simulate.py [hunk 1]
```diff
@@ -587,15 +587,27 @@ def adjacency_to_root_dfs(
...
-    params: dict,
+    params: CellSimulationParams,
     rng: interface.JAXRandomKey,
 ) -> dict:
     """
     Generates cell mutation matrix for one tree.
 
     Args:
-        params: input parameters from parser
+        params: TypedDict from parser of cell_simulation.py
```
Comment (gordonkoehn) [inline src/pyggdrasil/tree_inference/_simulate.py:610]: Reading this code again - now with the knowledge about `TypedDict`s - I felt the urge to do at least a little bit of type checking/enforcing parameters here.

**Review by pawel-czyz — APPROVED** [review]
Wonderful! Please, merge :slightly_smiling_face:

File: src/pyggdrasil/tree_inference/_simulate.py (hunk 1 above)
Comment (pawel-czyz) [reply src/pyggdrasil/tree_inference/_simulate.py]: For validation I really recommend [Pydantic](https://docs.pydantic.dev/latest/usage/models/), it's really excellent.

Comment (gordonkoehn) [reply src/pyggdrasil/tree_inference/_simulate.py]: Added a Pydantic model now :)

**Pull request merged**